ballerina-shell/modules/shell-core/src/main/java/io/ballerina/shell/snippet/factory/BasicSnippetFactory.java

```diff
@@ -157,6 +163,11 @@ public VariableDeclarationSnippet createVariableDeclarationSnippet(Node node) {
     @Override
     public ModuleMemberDeclarationSnippet createModuleMemberDeclarationSnippet(Node node)
             throws SnippetException {
+        if (isContainsIsolated(node)) {
```

Comment:
Instead of `isContainsIsolated`, shall we use `containsIsolated`?

Reply:
Updated in 3a77f2123f76ad957c11b56f10d1efecda8bca08